test/parallel/test-async-wrap-constructor.js

```diff
@@ -6,14 +6,14 @@ require('../common');
 const assert = require('assert');
 const async_hooks = require('async_hooks');
 
-for (const badArg of [0, 1, false, true, null, 'hello']) {
+[0, 1, false, true, null, 'hello'].forEach((badArg) => {
   const hookNames = ['init', 'before', 'after', 'destroy', 'promiseResolve'];
-  for (const field of hookNames) {
+  hookNames.forEach((field) => {
     assert.throws(() => {
       async_hooks.createHook({ [field]: badArg });
     }, {
       code: 'ERR_ASYNC_CALLBACK',
       name: 'TypeError',
```

Comment:
I think we can add message assertion too.

```suggestion
      name: 'TypeError',
      message: `something.${field} must be a function`
```

Reply:
I agree with this

Reply:
I agree, thanks @juanarbol !